aip/aog/3010.md

```diff
@@ -0,0 +1,55 @@
+---
+aip:
+  id: 3010
+  state: draft
+  created: 2020-01-14
+  updated: 2020-01-14
+  type: guidance
+  scope: aog
+js:
+  - /assets/js/graphviz/viz.js
+  - /assets/js/graphviz/lite.render.js
+  - /assets/js/aip/aip-graphviz.js
+permalink: /aog/3010
+---
+
+# Actions on Google Vertical Integration Webhook Format
+
+If an action is part of a vertical program that uses Google Fulfillment, it
+usually needs to expose a webhook that Google will call to execute user queries
+via a structured protocol. This AIP defines requirement for the webhook format.
+
+## Guidance
+
+### Base URL in Action Package
+
+The `baseUrl` configured in Action Package defines the common part of the
+actual execution URL of the webhook. The base URL **may** include a trailing
+slash, but **must** work if `:customMethodName` is appended directly. The
+endpoint **must** be HTTPS and **must** have valid certificate.
+
+Examples:
+
+- `https://us-central1.cloudfunctions.net/myWebhook/`
```

Comment:
Should we also have a counterexample of a Firebase cloud function URL missing the trailing slash, as an example of a common error?

Reply:
Done